lib/bindings/http/receiver_binary.js

```diff
@@ -52,16 +53,12 @@ BinaryHTTPReceiver.prototype.check = function(payload, headers) {
   // Clone and low case all headers names
   const sanityHeaders = Commons.sanityAndClone(headers);
 
-  // If no content type is provided, default to application/json
-  if (!sanityHeaders[Constants.HEADER_CONTENT_TYPE]) {
-    sanityHeaders[Constants.HEADER_CONTENT_TYPE] = Constants.MIME_JSON;
-  }
-
-  // Validation Level 1
-  if (!this.allowedContentTypes
-    .includes(sanityHeaders[Constants.HEADER_CONTENT_TYPE])) {
+  // Validation Level 1 - if content-type exists, be sure it's
+  // an allowed type
+  if (sanityHeaders[HEADER_CONTENT_TYPE] && !this.allowedContentTypes
```

Comment:
It would be nice to simplify that part to be a little better readable. But not required. 
The issue here is the wording + the `!` 
If ( foo is --> not_this_allowed_includes )

But at the same time I don't have a good suggestion for this... maybe this bellow ?
```
  const isNotIncludedHeader = !this.allowedContentTypes
    .includes(sanityHeaders[HEADER_CONTENT_TYPE]);

  if (sanityHeaders[HEADER_CONTENT_TYPE] && isNotIncludedHeader) {
```

Reply:
Done. Thanks for the suggestion.